src/selection.cpp

```diff
@@ -119,6 +118,14 @@ Selection operator|(const Selection& lhs, const Selection& rhs) {
     return detail::union_(lhs.ranges(), rhs.ranges());
 }
 
+bool Selection::contains(Value gid) const {
+    auto it =
```

Comment:
note that there isn't an invariant that a selection is sorted, so something like:
```
s=libsonata.Selection(((100, 200), (0, 10)))
0 in s -> False
1 in s -> False
```
Seems incorrect.

Reply:
it was done on purpose. Thank you for reminding me:

in node_sets we use maps and sets. Thus, everything is and always be sorted. We can even see the effects: if we put them unsorted (as you can see my test in compartment_sets.json is on purpose reverse-sorted) we get a sorted result. 

TLDR: I wanted to keep consistency with node_sets. It is an undocumented feature (due to the underlying implementation). What do you want to do?

Reply:
This is in Selection though, which doesn't have any consistency with `node_sets`; it's way of compactly storying `node_id`s.

As such, if `contains` is implemented, I would expect it to behave correctly; ie: say whether a node_id is in a selection, even if it's unsorted.

Reply:
I will add a test in any case to check this

Reply:
That's been thought about before, but sometimes someone already has an unsorted list of ids they want, and they are using it to get properties, and they want the properties returned in the same order.

For now it will just have to be less efficient, which is fine.